firmware/arduino/chessboard/chessboard.ino

```diff
@@ -97,6 +98,9 @@ void setup()
 
   stepsPerUnitSpace = MILLIMETERS_PER_UNITSPACE * STEPS_PER_MILLIMETER;
 
+  // Calculates the pulses needed to make circles for centerPiece()
+  calculatePulsesPerSlope();
```

Comment:
@nashirj, This defines the pulsesPerSlope array in circleFunction.ino.
What are your thoughts about defining it in chessboard.ino and making this function return pulsesPerSlope.
I'm thinking the call could look something like this:
`
pulsesPerSlope = calculatePulsesPerSlope();
`
Even though the assignment is functionally pointless since the array is global, it would better indicate where the data is being stored.

Reply:
I think if the comment above the function said something like
```
// Initializes global 2d array `pulsesPerSlope` which is used to define circle paths
// and used in the `centerPiece()` function in circleFunction.ino
```
then it would be clear and you wouldn't need to have a fake function call type deal